### DISCUSSION TITLE
How do I make it read from my local uncommited local files?

### DISCUSSION THREAD

**Discussion by lordvcs** [firecow/gitlab-ci-local#828] [Q&A] [answered]
Every time I run `gitlab-ci-local <jobname>`, it seems to pull the gitlab repo and run on the files available in the repo; which means when I, for eg, add a new file in my local(not yet committed and pushed to gitlab) and my job depends on the new file, it fails because the file is not available to `gitlab-ci-local`. 

So my question is, is it possible to make gitlab-ci-local read my current local uncommitted files when running a job?

BTW, Thanks for making this, super userful

**Comment by firecow** [comment, marked as answer]
Yes, but you need to git add them
https://github.com/firecow/gitlab-ci-local#tracked-files

**Reply by lordvcs** [reply]
Sorry for the late reply but this works great(I read the whole documentation but somehow missed this). Thanks for making this project, helped me save hours of unnecessary work
